device/vxlan/unicast.yaml

```diff
@@ -0,0 +1,45 @@
+components:
+  schemas:
+    Vxlan.Unicast:
+      description: >-
+        Configuration of Unicast parameters for VxLAN.
+      type: object
+      properties:
+        remote_vtep_address:
+          description: >-
+            Remote VXLAN Tunnel End Point address
+          $ref: '#/components/schemas/Remote.Vtep.Address'
+        suppress_arp:
+          description: >-
+            Supress ARP 
+          type: boolean
+          default: false
+        remote_vm_mac:
+          description: >-
+            Remote VM MAC addresses
+          type: array
+          items:
+            type: string
+            format: mac
+        remote_vm_ipv4:
```

Comment:
unicast infos not required,if required remote_vm_ipv6 should also be present

Reply:
Since this is not a breaking change, we can add this later based on user's requirement